tests/testIndexReference.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
There is already a test that tests some of the functionality of the indexer.  It might be worthwhile thinking about combining them.